package.json

```diff
@@ -22,7 +22,7 @@
     "test:regressions": "webpack --config test/regressions/webpack.config.js && rm -rf test/screenshots && happo run && babel-node ./test/regressions/normalize.js",
     "test:size": "bundlesize",
     "argos": "argos upload test/screenshots --token $ARGOS_TOKEN || true",
-    "dev": "jest --watch --bail",
+    "dev": "jest --watch --bail --projects packages/*",
```

Comment:
I provide the `projects` option to the CLI because right now there is an issue with Jest when the projects target only one package. I'll remove this when the package will be split.